_dashboards/discover/index.md

```diff
@@ -1,17 +1,94 @@
 ---
 layout: default
-title: Discover
-nav_order: 10
+title: Exploring data with Discover
+nav_order: 20
 has_children: true
 ---
 
-# Discover 
+# Exploring data with Discover 
 
-**Discover** in OpenSearch Dashboards helps you extract insights and get value out of data assets across your organization. You can quickly ingest and query your data, display that data in visualizations and interactive dashboards, and deliver insights to your organization.
+**Discover** in OpenSearch Dashboards helps you extract insights and get value out of data assets across your organization. Discover enables you to:
 
-<img src="{{site.url}}{{site.baseurl}}/images/discover-index.png" alt="Discover user interface">
+1. **Select data.** You can explore, customize, and filter data, as well as search data using the [Dashboards Query Language (DQL)]({{site.url}}{{site.baseurl}}/dashboards/dql/).
+2. **Explore the data.** You can analyze data details, view individual documents, and create tables summarizing data contents.
+3. **Visualize data.** You can display findings in a single dashboard showing different data visualization types.
 
-## What's next?
+## Try it: Exploring sample data with Discover
 
-- [Running queries in the console]({{site.url}}{{site.baseurl}}/dashboards/run-queries/)
-- [Creating a dashboard with multiple data sources]({{site.url}}{{site.baseurl}}/)
+This tutorial shows you how to use Discover to analyze and understand a sample data set. At the end of this tutorial, you should be ready to use Discover with your own data.
+
+Before continuing this tutorial, make sure you've added the **Sample flight data**. See [OpenSearch Dashboards Quickstart]() for more details on getting started.
+{: .warning}
+### Setting up data
+
+![Setting up the sample data in Discover]({{site.url}}{{site.baseurl}}/images/discover-setting-up-data.gif)
+
+1. Verify access to OpenSearch Dashboards by connecting to [http://localhost:5601](http://localhost:5601) from a browser. The default username and password are `admin`. 
+2. On the **Home** page, select **Discover** in the navigation pane.
+3. On the index pattern toolbar, select the **opensearch_dashboards_sample_data_flights** data set.
+4. On the time filter toolbar, select the calendar icon and change the time range to **Last 7 days**.
+
+### Exploring the data fields
+
+In the Discover panel, you'll see a table that shows all the documents that match your search. The table includes a list of data fields that are available in the document table.
+
+1. View the list of **Available fields**.
+1. Select **Cancelled** to view the **Top 5 Values**.
```

Comment:
I think `Cancelled` only has two values: true and false. 
We could change to a diff variable, like `DestCountry`.

Reply:
Change made.